### PR TITLE
fix(l10n): remove hardcoded string, remove capitalize classname

### PR DESCRIPTION
**Pull request by wescopeland**
Addresses feedback from https://discord.com/channels/310192285306454017/1299083037908406402/1305276501729742859.

### CODE REVIEW

File: resources/js/features/game-list/components/DataTableFacetedFilter/DataTableFacetedFilter.tsx [hunk 1]
```diff
@@ -82,14 +82,14 @@ export function DataTableFacetedFilter<TData, TValue>({
           size="sm"
           className={cn(
             'border-dashed',
-            buildTrackingClassNames(`Click ${title} Filter`),
+            buildTrackingClassNames(`Click ${t_title} Filter`),
```
Comment (Jamiras) [inline resources/js/features/game-list/components/DataTableFacetedFilter/DataTableFacetedFilter.tsx:85]: For metrics tracking, shouldn't this use an untranslated string?

File: resources/js/features/game-list/components/DataTableFacetedFilter/DataTableFacetedFilter.tsx [hunk 2]
```diff
@@ -82,14 +82,14 @@ export function DataTableFacetedFilter<TData, TValue>({
...
             className,
           )}
-          data-testid={`filter-${title}`}
+          data-testid={`filter-${t_title}`}
```
Comment (Jamiras) [inline resources/js/features/game-list/components/DataTableFacetedFilter/DataTableFacetedFilter.tsx:88]: This also seems like something that might be better off using a non-translated string. Perhaps even a string without spaces.

File: resources/js/features/game-list/components/DataTableFacetedFilter/DataTableFacetedFilter.tsx (hunk 1 above)
Comment (wescopeland) [reply resources/js/features/game-list/components/DataTableFacetedFilter/DataTableFacetedFilter.tsx]: Good call - in latest I've changed this to use the column ID instead.

File: resources/js/features/game-list/components/DataTableFacetedFilter/DataTableFacetedFilter.tsx (hunk 2 above)
Comment (wescopeland) [reply resources/js/features/game-list/components/DataTableFacetedFilter/DataTableFacetedFilter.tsx]: Agreed, I've also changed this in latest to now use the column ID.

**Review by Jamiras — APPROVED** [review]

**Pull request merged**